pkg/api/v1/helper/helpers.go

```diff
@@ -433,6 +433,26 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) string {
```

Comment:
Why it returns string and not `PersistentVolumeMode`?

Reply:
ack, I changed and moved this to the API change